gui/wxpython/core/ws.py

```diff
@@ -249,17 +249,17 @@ def _fitAspect(self, region, size):
 
             center = (region["east"] - region["west"]) / 2
 
-            region["east"] = center + delta + region["west"]
-            region["west"] = center - delta + region["west"]
+            region["east"] = min(center + delta + region["west"], 180.0)
+            region["west"] = max(center - delta + region["west"], -180.0)
```

Comment:
I can't really figure out where the values are coming from. Is this always in degrees/LL?

Reply:
Hmm... that's a good question. I don't think it's always in degrees.. I think the values are coming from:
https://github.com/OSGeo/grass/blob/2bb77ed6a481cf5e82ec323a6d9ee4dacbd306f3/gui/wxpython/core/render.py#L1105

so.. what we really need is the min/max coordinates of the projection here. I'll close this PR for now since it's not a good fix.

Reply:
You could also check if the projection is LL, that's usually the test performed elsewhere.

Reply:
I think we just need this fix for LL locations?